oci/spec_opts_windows_test.go

```diff
@@ -56,3 +56,57 @@ func TestWithCPUCount(t *testing.T) {
 		}
 	}
 }
+
+func TestWithWindowsIgnoreFlushesDuringBoot(t *testing.T) {
```

Comment:
Do you want a negative test against these to know they weren't just set by default?

Reply:
> Do you want a negative test against these to know they weren't just set by default?

Put another way "Justin would like a negative test against these...." ;) Sure, I'll add.

Reply:
LOL

Reply:
As mentioned offline, https://play.golang.org/p/3j9NNbLyZbx. So I have to have at least one option otherwise the tests would panic. But I don't think this is particularly important to verify.

Reply:
Lol. No... Your playground link is borked. For variadic args if you want "no options" you need to pass none. Look here: https://play.golang.org/p/V91bbaYfUyy

BUT: either way I am ok with you not doing this test.